example/README.md

```diff
@@ -1,8 +1,38 @@
 # intercom_example
 
-Demonstrates how to use the intercom plugin.
+Demonstrates how to use the intercom plugin in a Flutter project. 
 
-## Getting Started
+## initialize & identify user
+
+see example code at [lib/main.dart](lib/main.dart)
+
+configure your app's main function as follows
+
+        void main() async {
```

Comment:
Wrap code blocks using Markdown's code formatter to properly format it.

1. Add ```dart before the code block
2. Insert the code block
3. Add ``` after the code block